<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSpeciesWidget</class>
 <widget class="QWidget" name="AddSpeciesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Species name* :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Genus* :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genusLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Subfamily :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subfamilyLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Family :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="familyLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cladeLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Clade* :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
